src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayOrganisationChecksController.cs

```diff
@@ -220,5 +220,41 @@ public async Task<IActionResult> EvaluateIcoNumberPage(IcoNumberViewModel viewMo
 
             return RedirectToAction("ViewApplication", "RoatpGateway", new { viewModel.ApplicationId });
         }
+
+        [HttpGet("/Roatp/Gateway/{applicationId}/Page/WebsiteAddress")]
+        public async Task<IActionResult> GetWebsitePage(Guid applicationId)
+        {
+            var username = _contextAccessor.HttpContext.User.UserDisplayName();
+            var viewModel = await _orchestrator.GetWebsiteViewModel(new GetWebsiteRequest(applicationId, username));
+            return View("~/Views/Roatp/Apply/Gateway/pages/Website.cshtml", viewModel);
```

Comment:
There's a variable GatewayViewsLocation in the gateway-submaster that you can use instead of the long path

Reply:
Long path replaced with variable